README.md

```diff
@@ -78,6 +78,7 @@ Awesome Wagtail [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38
 - [Wagtail EditorJS](https://github.com/Nigel2392/wagtail_editorjs) - An [EditorJS](https://editorjs.io/) widget with great support for Wagtail's page, image and document choosers.
 - [Wagtail Terms](https://github.com/smark-1/wagtailterms) - A plugin to add a glossary terms entity to the Draftail editor.
 - [Wagtail Text Alignment](https://github.com/Nigel2392/wagtail_text_alignment) - Align text, headings and more in your Wagtail richtext editor.
+- [wagtailmdx]([link](https://github.com/TheBestDestroyer/wagtailmdx)) - A [MDXEditor](https://github.com/mdx-editor/editor) integration for Wagtail as textfield widget.
```

Comment:
```suggestion
- [wagtailmdx](https://github.com/TheBestDestroyer/wagtailmdx) - A [MDXEditor](https://github.com/mdx-editor/editor) integration for Wagtail as textfield widget.
```

Link needs fixing. 

Also, it would be good to see a some rest coverage on the repo, are you planning to add some?

Reply:
I can, but honestly I am not really sure what and how to test the integration of the editor? You have some suggestions which tests could be viable in this regard?

Reply:
OK I have taken another look, it's a pretty straight forward widget wrapper.

Maybe just a test that checks the field includes the right CSS/JS when used is sufficient.

I will approve this change now though, thanks for making this package.

Be sure to share in the Slack channel `#twiw` (this week in Wagtail) or in the watercooler chat.